dev/com.ibm.ws.concurrent.mp.cdi/src/com/ibm/ws/concurrent/mp/cdi/ConcurrencyCDIExtension.java

```diff
@@ -271,9 +272,23 @@ public void processInjectionPoint(Class<? extends Annotation> configAnnoClass, P
                 } else { // It is an error if 2 or more InjectionPoints define @NamedInstance("X") @ManagedExecutorConfig/ThreadContextConfig
                     if (instance instanceof ManagedExecutor)
                         ((ManagedExecutor) instance).shutdownNow();
-                    String msg = "ERROR: Found existing bean with name=" + instanceName; // TODO NLS message
-                    Tr.error(tc, msg);
-                    deploymentErrors.add(new Throwable(msg)); // TODO proper exception class
+
+                    ArrayList<String> conflicting = new ArrayList<String>();
+                    conflicting.add(injectionPointName);
+                    int index = 0;
+                    for (String name : instances.keySet()) {
+                        if (instanceName.equals(name)) {
+                            conflicting.add(injectionPointNames.get(index));
+                            break; // key can only appear once in set
+                        }
+                        index++;
+                    }
+
+                    String msg = Tr.formatMessage(tc, "CWWKC1190.duplicate.namedinstance",
```

Comment:
It looks like you added a new `CWWKC1190` message here, but I'm not finding it anywhere?  Did you forget to add a .nlsprops file to this commit? That would explain why you reduced the message range from 99 89 above

Reply:
Thanks for catching that. It is added now